server/sonar-web/src/main/js/apps/coding-rules/components/RuleDetailsMeta.tsx

```diff
@@ -125,6 +125,7 @@ export default class RuleDetailsMeta extends React.PureComponent<Props, State> {
         ) : (
           <TagsList
             allowUpdate={canWrite}
+            className="display-flex-center"
```

Comment:
could you also fix the display in the workspace as well please?

![image](https://user-images.githubusercontent.com/2317341/36585486-57010dce-187e-11e8-9624-c4af672a7d61.png)

Reply:
Ah I didn't see this, I'll fix it